dotcom-rendering/src/components/DecideContainer.tsx

```diff
@@ -221,7 +222,11 @@ export const DecideContainer = ({
 		case 'nav/media-list':
 			return <NavList trails={trails} showImage={true} />;
 		case 'fixed/highlights':
-			return <HighlightsContainer trails={trails} />;
+			return (
+				<Island priority="feature" defer={{ until: 'visible' }}>
+					<HighlightsContainer trails={trails} />
+				</Island>
+			);
```

Comment:
because we're using buttons we need the container to be an island for interactivity 🖱️